cli/launchpad/generic.go

```diff
@@ -0,0 +1,106 @@
+package launchpad
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+)
+
+const indentSize = 2
+
+var (
+	errValidationFailed = errors.New("validation failed")
+	tfNameRegex         = regexp.MustCompile("^[a-zA-Z][a-zA-Z\\d\\-\\_]*$")
+)
+
+//
+type resourcer interface {
```

Comment:
We shouldn't call this `resourcer` as `resource` is not a verb. Perhaps `ResourceHandler`?